modules/ROOT/pages/policies-compare-mule3-and-mule4.adoc

```diff
@@ -0,0 +1,41 @@
+= Policies and Mule Versions
```

Comment:
More precisely, "Comparison of Policy Features Between Mule 3 and Mule 4"

However, the items listed in the Features column are not all worded as a feature: an item or capability. Try to make the entries in the Features column parallel grammatically.

I also do not understand the empty table cells. If there is no text entry, please supply "Not applicable."

Reply:
Fixed! Thanks!